src/update_step.jl

```diff
@@ -1,12 +1,12 @@
-function tdvp_step(
+function update_step(
   order::TDVPOrder, solver, PH, time_step::Number, psi::MPS; current_time=0.0, kwargs...
 )
   orderings = ITensorTDVP.orderings(order)
   sub_time_steps = ITensorTDVP.sub_time_steps(order)
   sub_time_steps *= time_step
   global info
   for substep in 1:length(sub_time_steps)
-    psi, PH, info = tdvp_sweep(
+    psi, PH, info = local_sweep(
```

Comment:
In analogy to the name `update_step` I think it makes sense to call it `update_sweep` but maybe that's too close to `sweep_update`?

Maybe we could call them `sweep_update_step` and `sweep_update_sweep`.

Reply:
I'm good with changing the name of this. Here's my only issue – we'd then have the following three functions:
`sweep_update`
`update_step`
`sweep_update_step`

which I think if I came back to a few months from now I'd have a hard time telling apart since the names are so similar!

I think another issue is that I slightly misunderstood the purpose of the tdvp_sweep/local_sweep function. I think now I'm seeing it is doing basically a single half sweep, correct?

Reply:
(This may be easier to hash out in person next week! It's both kind of important long term but also not too important short term.)

Reply:
(Resolved after discussing with Matt and selecting `alternating_update` in line with the mathematics terminology. For example see https://en.everybodywiki.com/Alternating_Optimization)